webapp/channels/src/components/advanced_text_editor/use_textbox_focus.tsx

```diff
@@ -31,13 +31,28 @@ const useTextboxFocus = (
     const shouldFocusRHS = useSelector(getShouldFocusRHS, () => true);
 
     const focusTextbox = useCallback((keepFocus = false) => {
+        const doBlur = () => {
+            textboxRef.current?.blur();
+        };
+
+        const doFocus = () => {
+            textboxRef.current?.focus();
+        };
+
+        const wait = (call: () => void) => {
+            // Add a delay to allow for any animation.
+            setTimeout(() => {
+                call();
+            }, 100);
```

Comment:
The only concern I have is that (as with any fix that uses a timer) this could be brittle. Is there an "after animation has ended" or using something like intersection observer to know when it's visible?

Reply:
Possibly! @hmhealey, do you have any suggestions? This fixes the immediate issue, so I'm content with some brittleness compared to the current behavior, but if the lift to do it "right" isn't too heavy that would be awesome too!

Would love to resolve this by end of day.